doc/reference/gerbil/expander/README.md

```diff
@@ -218,11 +219,58 @@ Please document me!
 Please document me!
 
 ### import-module
+
 ```
-(import-module ...)
+(import-module path (reload? #f) (eval? #f))
 ```
 
-Please document me!
+This is the runtime functional way to import a module. 
+
+Unlike the `import` syntax this does not modify the
+[current-expander-context](###current-expander-context).
+
+*example*:
+
+We import the `:std/format` module context. That does not place `format` in our expander context's namespace. 
+
+ 
+  * `reload? #f` : We don't load it again if already imported.
+  * `eval? #t` : If not imported, evaluate the module source.
+  
+Because there may be other reasons to import a module but not have it `eval`'d, like for example finding dependencies and other prerequisites for compilation, there must be that argument. 
+
+``` scheme
+> (import :gerbil/expander :std/sugar)
+> (def ctx (import-module ':std/format #f #t))
+> (ignore-errors (format "~a" ctx))
+#f
+```
+
+Because that context was evaluated we now have runtime access to all the symbols in it.
+
+``` scheme
+> (def sym (gxc#find-runtime-symbol ctx 'format))
```

Comment:
using a compiler symbol without importing the appopriate module is a very bad idea...  guaranteed crash in a binary.